chapter1.md

```diff
@@ -134,7 +134,7 @@ kolla-genpwd
 ```
 
 ### Prepare Kolla config
-Now let's prepare actual config. Use your favorite editor to edit /etc/kolla/globals.yaml
+Now let's prepare actual config. Use your favorite editor to edit /etc/kolla/globals.yml
```

Comment:
Same goes for `passwords.(yml|yaml)`, see https://github.com/shdowofdeath/kolla-ansible-workshop/commit/f1e7939be6d84a329f679861e315ed11d52e2ba0